pithos/plugins/notification_icon.py

```diff
@@ -117,20 +153,20 @@ def set_active(active):
         self.menu = menu
 
     def play_state_changed(self, window, playing):
-        """ play or pause and rotate the text """
+        ''' play or pause and rotate the text '''
```

Comment:
No I changed them all at once the 1st time around and haven't changed them back yet,lol!!!